node-gui/src/main_window/main_widget/tabs/wallet/stake.rs

```diff
@@ -61,59 +61,72 @@ const CREATE_STAKING_POOL_TOOLTIP_TEXT: &str = "A staking pool locks the pledge
 const START_STAKING_TOOLTIP_TEXT: &str =
     "If you have created pools, this will activate staking. The node must be kept running, in order to assist the network in creating blocks and earn rewards.";
 
+const DECOMMISSION_POOL_ID_TOOLTIP_TEXT: &str = "The pool ID to decommission.";
+const DECOMMISSION_POOL_ADDRESS_TOOLTIP_TEXT: &str =
+    "The output address where the coins will be sent after decommissioning the pool.";
+const DECOMMISSION_POOL_TOOLTIP_TEXT: &str =
+    "This will decommission the pool and lock the pledged amount for 720 blocks.";
```

Comment:
"This will decommission the staking pool and lock the pledge amount for the maturity period (see above to learn how long). Locking implies that the coins cannot be moved until the lock period is over. This is needed for blockchain security reasons. Please STOP STAKING before doing this, otherwise decommissioning won't work. DO NOT use an exchange address for this. Most exchanges cannot handle locked outputs."

It's actually 7200 blocks, not 720... this is why I never wrote this info manually, to ensure these mistakes never happen.